### PR TITLE
Updated README and remove obsolete files

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+Eclipse Xtext is a framework for development of programming languages and [domain specific languages](https://en.wikipedia.org/wiki/Domain-specific_language). It covers all aspects of a complete language infrastructure, from parsers, over linker, compiler or interpreter to fully-blown top-notch IDE integration for Eclipse and Web-based IDEs. Further, languages developed with Xtext can be integrated into editors supporting the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) (e.g. [VS Code](https://code.visualstudio.com/), [Eclipse Che](https://www.eclipse.org/che/)). Xtext comes with great defaults for all these aspects, which at the same time can be easily tailored to your individual needs.
+
+More information can be found on [xtext.org](http://xtext.org).
+
+## Quick Access
+
+- [All Open Pull Requests](https://github.com/search?utf8=✓&q=is%3Apr+is%3Aopen+repo%3Aeclipse%2Fxtext+repo%3Aeclipse%2Fxtext-umbrella+repo%3Aeclipse%2Fxtext-core+repo%3Aeclipse%2Fxtext-lib+repo%3Aeclipse%2Fxtext-extras+repo%3Aeclipse%2Fxtext-eclipse+repo%3Aeclipse%2Fxtext-web+repo%3Aeclipse%2Fxtext-maven+repo%3Aeclipse%2Fxtext-xtend&type=Issues&ref=searchresults)
```
Comment (cdietrich) [inline README.md:11]: TODO do we want to update these queries just to query the 2 active repos or keep it as is?

File: dev-doc/MaintenanceTasks.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Maintenance Tasks
+
+This document describes things that need to be done during a simrel.
+
+## Bump versions
```
Comment (cdietrich) [inline dev-doc/MaintenanceTasks.md:5]: TODO: update this to the new mechanism

File: dev-doc/MaintenanceTasks.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+We build `xtext-eclipse` and `xtext-xtend` against the latest i-builds of platform and the latest milestone of the rest of the simrel. Thus with every version we have to adapt the target platforms.
+
+* In [xtext-eclipse](https://github.com/eclipse/xtext-eclipse)
+  * In `releng/org.eclipse.xtext.target/org.eclipse.xtext.target-latest.target` use the latest i-build `https://download.eclipse.org/eclipse/updates/<number>-I-builds` and simrel `https://download.eclipse.org/releases/<version>`
+* In [xtext-xtend](https://github.com/eclipse/xtext-xtend)
+  * In  `releng/org.eclipse.xtend.target/org.eclipse.xtend.target-latest.target` use the latest i-build `https://download.eclipse.org/eclipse/updates/<number>-I-builds` and simrel `https://download.eclipse.org/releases/<version>`
+* The simrel `https://download.eclipse.org/releases/<version>` needs also to be used in `./xtext-eclipse/org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/tp/domainmodel.target` and `xtext-core` in `TargetPlatformProject` +  tests adapted (`CliWizardIntegrationTest`)
+
+## Update Orbit
```
Comment (cdietrich) [inline dev-doc/MaintenanceTasks.md:51]: outdated

File: dev-doc/RegenerateLanguages.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Regenerate Languages
+
+This document describes the necessary steps to regenerate the test- and production Xtext languages.
+
+1. Execute the following `MWE2` workflows from Eclipse
```
[the remark this answers is not in the document]
Comment (szarnekow) [reply dev-doc/RegenerateLanguages.md:5]: It will be replaced by a launch group. At least I hope so.

**Comment by LorenzoBettini** [comment]
Concerning `fixVersions.sh`:

I forgot to mention that I worked on a few script files, which I haven't committed yet, that only run a Maven command relying on Maven/Tycho version plugins. These take care of updating the version consistently.

However, if you have already experimented with this script file and it works, we could stick with it now.

**Comment by HannesWell** [comment]
Besides the already removed files I wonder if the following files are obsolete as well:
- org.eclipse.xtext.contributor/component.def
- org.eclipse.xtext.contributor/workspace.epf
- tyho-test.sh

**Comment by cdietrich** [comment]
@szarnekow @LorenzoBettini what is the state of this pr?

**Comment by LorenzoBettini** [comment]
@cdietrich I could use this issue to create/update the release instructions/details?

**Comment by cdietrich** [comment]
@LorenzoBettini  yes but i am not sure if the state is current from @szarnekow

File: README.md (hunk 1 above)
Comment (szarnekow) [reply README.md:11]: I'll update the queries

File: dev-doc/MaintenanceTasks.md (hunk 2 above)
Comment (szarnekow) [reply dev-doc/MaintenanceTasks.md:5]: I'll leave this for @LorenzoBettini

**Comment by szarnekow** [comment]
@LorenzoBettini, I'd prefer this to be merged to update the readme already. The documentation of the release procedure can happen on top of this. What do you think?

**Comment by LorenzoBettini** [comment]
> @LorenzoBettini, I'd prefer this to be merged to update the readme already. The documentation of the release procedure can happen on top of this. What do you think?

@szarnekow I agree! Please, go right ahead

**Pull request merged**